### ISSUE TITLE
MC-PCA

### ISSUE THREAD

**Issue by CommonClimate** [LinkedEarth/Pyleoclim_util#169]
**Is your feature request related to a problem? Please describe.**
Monte-Carlo PCA is presently disabled. We need a working one.

**Describe the solution you'd like**
We need:
- [x] data structures that can accommodate multivariate ensembles (e.g. MultipleSeries made of EnsembleSeries objects)
- [x] working, robust code, built on https://www.statsmodels.org/stable/generated/statsmodels.multivariate.pca.PCA.html
- [x] a `screeplot()` function with eigenvalue quantiles derived from the age ensemble
- [x] a `modeplot()` function that can map the spatial loadings, which involves resolving #167
- [x] a trivial test (can repurpose tests for the `pca()` method)
- [ ] an interesting test, e.g. reproduce Tierney et al Nature 2013 with these records:

<img width="767" alt="Screen Shot 2021-09-29 at 09 44 45" src="https://user-images.githubusercontent.com/13760223/135312876-ff5f2e88-3aba-4aab-8967-22470026c0d3.png">
(this would become a PaleoHack notebook)

Could be incorporated into PCA or be a separate function. 

**Describe alternatives you've considered**
leave things as is and let users work around the existing `pca()` function by looping over ensembles.

**Comment by victoriatodd2** [comment]
Is this feature available yet?

**Comment by CommonClimate** [comment]
Not yet - I was stuck on the first item, but @khider thinks it can be done with the `LiPDSeries` class. It's on the menu after version 1.0.0 is published, once the [paper](https://www.essoar.org/doi/abs/10.1002/essoar.10511883.1) is out. If you have a more pressing need, let us know and we can collaborate!

**Comment by alexkjames** [comment]
We have some code for doing this, it will be implemented soon, waiting on #525

**Comment by khider** [comment]
Will be tested on a Project Pythia cookbook

**Comment by khider** [comment]
Last bullet is a duplicate of  https://github.com/LinkedEarth/PaleoBooks/issues/11
